### ISSUE TITLE
When Using TalkBack Feature the Work With Us Is Read Inccorectly

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-android#7405]
**Describe the bug**
When TalkBack is reading over Work WIth Us it is read over as Work With U-S, instead of the word us, this should be fixed if possible. 

**To Reproduce**
1. Go to Menu
2. Click on Settings
3. Scroll down to Woo Commerece
4. Select Woo Commerece
5. Activate TalkBack
6. Select the Work With Us
7. Notice how it reads as Work With U-S

**Screenshots**
<img src="https://user-images.githubusercontent.com/36495451/190717772-601b5214-3755-41b2-beee-bab44afc6057.jpg" width="300"/>

**Expected behavior**
This should be read as Work With Us (Us the word)

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Samsung Galaxy A51
- Android version: 12
- WooCommerce Android version: 10.1
